src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -179,6 +179,22 @@ def _download_symbolic_link(dxid, md5digest, project, dest_filename):
                                             duration=6*3600,
                                             project=project)
 
+    def call_cmd(cmd, retry=True):
```

Comment:
Our convention elsewhere in the toolkit is to name it `ok_to_retry`.

Reply:
I've implemented the "exponential backoff" from the other comment. Is this fine?

Reply:
Looks good, I'd only set the max_retries to e.g. 4.

Reply:
Please also address this before merging. @mhrvol

Reply:
I checked email with this notification after I checked/merged this PR. There is another PR with this change. Feel free to merge after approving.